test/clojure_for_js_devs/handlers_test.clj

```diff
@@ -1,8 +1,14 @@
 (ns clojure-for-js-devs.handlers-test
   (:require [clojure.test :refer [deftest is testing]]
-            [clojure-for-js-devs.handlers :as handlers]))
+            [clojure-for-js-devs.handlers :as handlers]
+            [bond.james :as bond]
+            [clojure-for-js-devs.redis :as redis]))
 
 (deftest handlers
   (testing "hello-world-handler"
     (let [response "howdy!"]
-      (is (= (handlers/hello-world-handler) response)))))
+      (is (= (handlers/hello-world-handler) response))))
+  (testing "counter-handler"
+    (let [response "Counter: 44"]
+      (bond/with-stub! [[redis/getKey (constantly 44)] [redis/incr (constantly nil)]]
+        (is (= (handlers/counter-handler {:remote-addr "1.2.3.4.5"} {}) response))))))
```

Comment:
Minor point but we could use ring-mocks for the request map. https://github.com/circleci/growth-auth-service/blob/main/test/circleci/growth_auth_service/handlers_test.clj#L23-L24